index.d.ts

```diff
@@ -1,6 +1,8 @@
 import { parsed } from "./lib/shared";
 import type { SemVer } from "semver";
+import type { RANGE_PATTERN } from "./lib/specifier";
 
+export type { RANGE_PATTERN };
```

Comment:
```suggestion
import { RANGE_PATTERN } from "./lib/specifier";
 
export { RANGE_PATTERN };
```
This needs real export, as we need to use the value

Reply:
In renovate we directly import value from @renovate/pep440/lib/specifier.

Reply:
I know, i like to change it because this don't looks nice:

![image](https://user-images.githubusercontent.com/1798109/148918317-97d465d1-d842-4c3f-a595-a119374e37fc.png)

😉